apps/ui/src/Pages/HomePage/HomePage.tsx

```diff
@@ -98,6 +101,19 @@ const HomePage: React.FC = () => {
             <ToggleButton value={'communities'}>My Communities</ToggleButton>
             <ToggleButton value={'rides'}>My Rides</ToggleButton>
           </ToggleButtonGroup>
+          {selectedTab === 'rides' && (
```

Comment:
Why did you also add it here?

It should only be in the rides list components

Reply:
@DarNachmani 
Removed from home page it's now only in ride list